README.md

```diff
@@ -43,6 +43,7 @@ Here's an example `brillo.yml` for IMDB:
 
 ```yaml
 name: imdb            # Namespace the scrubbed file will occupy in S3
+compress: true
```

Comment:
> Is this not the default behaviour? I don't see a need to put it in the example if it is. Convention over configuration and all that

 I added it to be able to document it as it is kind of hidden away. I found it while I was building the other config variables and noticed it was useful and I really didn't knew about it since It's not mentioned anywhere else on the readme.